Tests/Application/config/swiftmailer.yml

```diff
@@ -0,0 +1,2 @@
+swiftmailer:
+    disable_delivery:  true
```

Comment:
One question, why do we have a swiftmailer config, if swiftmailer isn't installed?

Reply:
its installed when using Symfony 5.4 or Sulu 2.4 (lower dependencies)

Reply:
and there its still the default mailer to not break projects which maybe did create swiftmailer listeners.

Reply:
handled here: https://github.com/sulu/SuluFormBundle/blob/ba31c3e69d477b5665e24ffb23d91bbd446e6edc/DependencyInjection/SuluFormExtension.php#L244-L258